testplan/common/utils/logger.py

```diff
@@ -80,9 +80,16 @@ def driver_info(self, msg, *args, **kwargs):
         """Log 'msg % args' with severity 'DRIVER_INFO'"""
         self._custom_log(DRIVER_INFO, msg, *args, **kwargs)
 
-    def log_test_status(self, name, passed, indent=0, level=TEST_INFO):
+    def log_test_status(self, name, passed,
```

Comment:
It's a bit odd that this function now takes both a required `passed` boolean and an optional `status` string. Why not just pass the `status` string?

Reply:
> It's a bit odd that this function now takes both a required `passed` boolean and an optional `status` string. Why not just pass the `status` string?

How to display the text color? Add a status color map?

Reply:
Yeah maybe, or just a set of PASSED_STATUSES and FAILED_STATUSES. You could use the same sets in the passed / failed property functions.